src/styles/_catalog.scss

```diff
@@ -1,22 +1,26 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  $media:
+    1px 1,
+    488px 2,
+    768px 3,
+    1024px 4;
```

Comment:
the map is a key and value pair. key and value. the first is key, the second is value